app/controllers/stages_controller.rb

```diff
@@ -145,4 +145,10 @@ def find_project
   def find_stage
     @stage = @project.stages.find_by_param!(params[:id])
   end
+
+  def update_deploy_groups
```

Comment:
Thanks for making me keep at this, I got the IDs working, and activerecord seems to handle the blank ID when using the deploy_group_ids assignment.